tools/build-agent-release.sh

```diff
@@ -0,0 +1,54 @@
+RELEASEDIR=releases
+TAG=$(date +%Y%m%d)-0.$(git log --pretty=format:'%h' -n 1)
```

Comment:
Consider some error checking of these two subcommands’ output, since this is a convenient time (at line 2) to detect if ‘git log’ isn’t working properly. (Unlikely, but if you do end up with a tag ending in “0.” someday, this is why!)